src/main/java/mate/model/Driver.java

```diff
@@ -1,57 +1,29 @@
 package mate.model;
 
-import java.util.Objects;
-
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@ToString
```

Comment:
if we use lombok tostring annotation driver's password will be also visible there, let's exclude it